packages/filters/filter-color-matrix/src/ColorMatrixFilter.ts

```diff
@@ -40,14 +42,14 @@ export class ColorMatrixFilter extends Filter
      * @param {boolean} multiply - if true, current matrix and matrix are multiplied. If false,
      *  just set the current matrix with @param matrix
      */
-    _loadMatrix(matrix, multiply = false)
+    private _loadMatrix(matrix: number[], multiply = false): void
```

Comment:
This class might benefit from a more strict matrix with a fixed length:

```typescript
// T stands for (content) type, L stands for the array fixed length
type ArrayFixed<T, L extends number> = [ T, ...Array<T> ] & { length: L }
type ColorMatrix = ArrayFixed<number, 20>;
```

Reply:
Ok added this!